site/src/index.html

```diff
@@ -0,0 +1,153 @@
+---
+layout: default
+externals:
+  index-example-js: index-example.js
```

Comment:
I don't think this is used. If it isn't this line **and the file** can be deleted